src/coldcard.ts

```diff
@@ -550,6 +529,18 @@ export class ColdcardSignMultisigTransaction extends ColdcardInteraction {
  *
  */
 export class ColdcardMultisigWalletConfig {
+  jsonConfig: any;
+
+  name: string;
+
+  requiredSigners: number;
+
+  totalSigners: number;
+
+  addressType: string;
+
+  extendedPublicKeys: any[];
```

Comment:
```suggestion
  extendedPublicKeys: WalletConfigKeyDerivation[];
```

from [here](https://github.com/unchained-capital/unchained-wallets/blob/master/src/types/index.ts#L20-L24).

Reply:
All of these constructor initialization values are being pulled from `jsonConfig`. Can the `jsonConfig` actually be definitely typed, or is the entire thing really an `unknown`? Is there any type validation for `jsonConfig`?

Reply:
Changed in 35668a8805582ecc715ddc5e5214ac4de91c22ff